### PR TITLE
Include docker compose 

### PR DESCRIPTION
**Pull request by smohiudd**
Include docker compose and Dockerfiles for local development, similar to `veda-backend`

### CODE REVIEW

File: local/Dockerfile.db [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM ghcr.io/vincentsarago/postgis:14-3.3
+
+#sample data
+COPY data/*.sql /docker-entrypoint-initdb.d/
```
Comment (ividito) [inline local/Dockerfile.db:4]: `docker compose up` fails for me on this line - is this a feature to seed a dev database, or is there a missing file in the PR?

Comment (smohiudd) [reply local/Dockerfile.db:4]: Yes that's just to include some sample data in the database. You an either comment out line 4 or include a `data` folder with some sample data. Can use this and try: https://github.com/developmentseed/tifeatures/tree/master/data

File: stack/app/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-timvt==0.8.0a2
-tifeatures==0.1.0a2
+timvt
+tifeatures
```
Comment (vincentsarago) [inline stack/app/requirements.txt:2]: knowing the person who maintains those 2 projects I would recommend to pin to specific version 😂

Comment (smohiudd) [reply stack/app/requirements.txt:2]: I had some issues with the build when including the versions. Let me try it again and troubleshoot...

File: local/Dockerfile.db (hunk 1 above)
Comment (smohiudd) [reply local/Dockerfile.db]: I removed COPY data. For local dev, we can load data using ogr2ogr

File: stack/app/requirements.txt (hunk 2 above)
Comment (smohiudd) [reply stack/app/requirements.txt]: I added versions to the requirements

**Review by ividito — APPROVED** [review]

**Pull request merged**